### DISCUSSION TITLE
Webui not accepting API info

### DISCUSSION THREAD

**Discussion by tuxydoh** [mrlt8/docker-wyze-bridge#1314] [Q&A] [answered]
First time installer, got it set up in docker, can launch the web ui, however the ui is not as described in the quick start. It is asking for full email and password, api ID and api Key. When I enter the ID info I get an error about it being 32 characters, I have tried both key and id in the field to make sure I wasnt putting them in backwards, but both return same error and I cannot proceed any further

**Comment by mrlt8** [comment, marked as answer]
hmm, can you try generating a new key on the dev portal?
https://developer-api-console.wyze.com/index.html

The Key Id should be 36 characters long.

**Reply by tuxydoh** [reply]
Resetting it seems to have worked. It allows me to log in now. Images are stills however and the stream feeds all show locks

**Comment by NewAiCoder** [comment]
Hey - I am having the same issue. No matter how many times I try to re-generate the api key, the Key Id is never 36 characters long - it's 37, thus wyze-bridge won't accept it :(

**Reply by NewAiCoder** [reply]
It worked when I entered my API credentials directly in docker compose, not the web UI - I am guessing there is a hard lock for 36 characters in the web ui configuration when it should actually be 37 (or no restriction at all for the input)

**Reply by Ohlysmokes** [reply]
I noticed with mine there was a leading space to be deleted but I still couldn't log in.
